### DISCUSSION TITLE
New Event: Use Item, like Equip Item but for Consumables or Other Event Items.

### DISCUSSION THREAD

**Discussion by Uyarrr** [AscensionGameDev/Intersect-Engine#1700] [Ideas]
New Event: Use Item, like Equip Item but for Consumables or Other Event Items. It will be really useful for developers. Example, you can give temporary buffs to players with events that way. You can give buffs or cast spells via events.

**Comment by WeylonSantana** [comment]
In that case wouldn't the best option be a give spell effect option to be more direct? then choose the effect, icon (from the spell folder) and duration. that way you could also have the option to remove
